### ISSUE TITLE
Need 1 (or 2) icons for #2289

### ISSUE THREAD

**Issue by bmarty** [vector-im/element-android#2351]
We need 2 new icons for what has been done in #2289: It is now possible to create a DM either by entering matrixId (the legacy way), or, new, by scanning a QR code.

The existing icons has been used for the moment, this is what we have:

<img width="333" alt="image" src="https://user-images.githubusercontent.com/3940906/98354044-7cd04b80-2020-11eb-86d7-202c211130e9.png">

Maybe only one new icon (for "Add by QR code") action will be enough.

**Comment by niquewoodhouse** [comment]
@bmarty Interesting idea - can definitely see that being very useful :) 

Do you think this might have a negative impact?  For example, it's not clear from the options that pop up how I can add people from my contact book.  I don't know what matrix ID means, and I don't want to use QR code, I just want to invite David in my contacts on my phone.  These two options might make it hard to know what to pick/do.

I'm sure its been considered but maybe it would be better to put it on the next screen, next to the other options at the top:

![Screenshot_20201106-134829](https://user-images.githubusercontent.com/1587166/98374424-8d8cbb80-2038-11eb-9b60-87ac9037031a.png)

I put a couple of QR icons [here](https://www.figma.com/file/KhVjYAzy4oX3eIqwu9KQxN?node-id=3372:0#48228595)

**Comment by TR-SLimey** [comment]
@niquewoodhouse That was what I did originally (see screenshots section in #2276) but we decided that it would be better to move it into the roomlist as (1) it behaved differently to the other buttons (instantly created the room as opposed to adding to the list) and (2) that screen is supposed to be getting a re-work. I think another positive of this is that it's quicker to access which is good in the scenarios it would be used in - face-to-face. To solve the problem you mentioned, maybe `Add by matrix ID` could just be replaced with `Search` or `Search for users`? WDYT?
